l10n_pt_partner/README.rst

```diff
@@ -0,0 +1,65 @@
+Portugal - Partners
+============================================================
```

Comment:
Please use the current version of the README template: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst